tenable/io/exclusions.py

```diff
@@ -59,7 +60,7 @@ def create(self, name, members, start_time=None, end_time=None,
                 The day of the month to repeat a **MONTHLY** frequency rule on.
                 The default is today.
             enabled (bool, optional):
-                Is the exclusion enabled?  The default is ``True``
+                enable/disable exclusion. The default is ``True``
```

Comment:
I believe this is still incorrect. Perhaps it should be "enable/disable the exclusion schedule. The default is ``True``". See #273.